www/index.html

```diff
@@ -156,27 +154,28 @@ <h3>Credits</h3>
+                    <br>This application is licensed under the GPL v3 license :
+                    <br>
                     <i>
-                    <br />Kiwix is free software: you can redistribute it and/or modify
-                    <br />it under the terms of the GNU General Public License as published by
-                    <br />the Free Software Foundation, either version 3 of the License, or
-                    <br />(at your option) any later version.
-                    <br />
-                    <br />Kiwix is distributed in the hope that it will be useful,
-                    <br />but WITHOUT ANY WARRANTY; without even the implied warranty of
-                    <br />MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-                    <br />GNU General Public License for more details.
-                    <br />
-                    <br />You should have received a copy of the GNU General Public License
-                    <br />along with Kiwix (file LICENSE-GPLv3.txt).  If not, see <a href="http://www.gnu.org/licenses/">http://www.gnu.org/licenses/</a>
+                    <br>Kiwix is free software: you can redistribute it and/or modify
+                    <br>it under the terms of the GNU General Public License as published by
+                    <br>the Free Software Foundation, either version 3 of the License, or
+                    <br>(at your option) any later version.
+                    <br>
+                    <br>Kiwix is distributed in the hope that it will be useful,
+                    <br>but WITHOUT ANY WARRANTY; without even the implied warranty of
+                    <br>MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+                    <br>GNU General Public License for more details.
+                    <br>
+                    <br>You should have received a copy of the GNU General Public License
+                    <br>along with Kiwix (file LICENSE-GPLv3.txt).  If not, see <a href="http://www.gnu.org/licenses/">http://www.gnu.org/licenses/</a>
                     </i>
-                    <br />
-                    <br />Main libraries and ressources used :
+                    <br>
+                    <br>Main libraries and ressources used :
                     <ul>
                         <li><a href="http://jquery.com/" target="_blank">jQuery</a> 3, released under the <a href="http://jquery.org/license" target="_blank">MIT license</a></li>
-                        <li><a href="http://getbootstrap.com/" target="_blank">Bootstrap</a> 3, released under the <a href="http://www.apache.org/licenses/LICENSE-2.0" target="_blank">Apache license 2.0</a>, which includes icons from <a href="http://glyphicons.com/">Glyphicons</a></li>
+                        <li><a href="http://getbootstrap.com/" target="_blank">Bootstrap 4</a>, released under an <a href="https://github.com/twbs/bootstrap/blob/master/LICENSE" target="_blank">MIT Licence</a></li>
```

Comment:
@kelson42 Do you mean this line (licence info)? If so, it's already in the PR, but the diff was hidden due to large number of (superficial) changes.

Reply:
@Jaifroid Yes, Sorry missed it.